File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
       int i=1;
       for(int j=1;j<nums.length;j++)
       {
        if(nums[i-1]!=nums[j])
        {
            nums[i]=nums[j];
            i++;
        }
       }
       return i;
     
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.c
int removeDuplicates(int* nums, int numsSize)
 {
    for (int i=0;i<numsSize;i++)
    {
        for(int j=i+1;j<numsSize;j++)
        {if(nums[i]!=999)
        {
            if(nums[i]==nums[j])
            {
                nums[j]=999;
            }
        }
        }
    }
    int k=0;
for(int i=0;i<numsSize;i++)
{
if(nums[i]==999)
continue;

nums[k]=nums[i];
k++;
}

return k;
}